src/training_hub/algorithms/lora_grpo.py

```
@@ -0,0 +1,1503 @@
+        os.environ["VLLM_USE_V1"] = "0"
+        try:
+            import art
+            from art.local.backend import LocalBackend
+            result = asyncio.run(
+                ARTLoRAGRPOBackend()._run_training(algorithm_params, art, LocalBackend)
+            )
+            with open(results_path, "w") as f:
+                json.dump(result, f, indent=2)
+        except SystemExit:
+            pass  # os._exit from shutdown — results already on disk
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)